### PR TITLE
Log status display changes

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
Changing the status displays creates an entry in game.log of who did it, and from where.
## Why It's Good For The Game
Seeing "LOL PENIS" on the status displays with no way to track it gets old after a while.
## Changelog
:cl: LT3
admin: Changing the station status displays now generates a log entry
/:cl:

### CODE REVIEW

**Comment by ZephyrTFA** [comment]
Not sure that I like it telling admins on every status change; given I don't believe there is any delay in doing so

**Comment by lessthnthree** [comment]
> Not sure that I like it telling admins on every status change; given I don't believe there is any delay in doing so

It's not a frequently occuring thing, and you can smite people who put up stupid things.

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -769,8 +769,14 @@
 		if("message")
 			status_signal.data["top_text"] = data1
 			status_signal.data["bottom_text"] = data2
+			message_admins("[key_name(usr)][ADMIN_LOOKUPFLW(usr)][ADMIN_SMITE(usr)] has changed the station status display message to \"[data1] [data2]\" [loc_name(usr)]")
```
Comment (Zonespace27) [inline code/game/machinery/computer/communications.dm:772]: This doesn't feel urgent enough to spam admins as opposed to the things that so (atmos, bombs, etc.)

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by lessthnthree** [comment]
Okay, I'll remove it.

File: code/game/machinery/computer/communications.dm [hunk 2]
```diff
@@ -769,8 +769,12 @@
 		if("message")
 			status_signal.data["top_text"] = data1
 			status_signal.data["bottom_text"] = data2
+			log_game("[key_name(usr)] has changed the station status display message to \"[data1] [data2]\" [loc_name(usr)]")
```
Comment (Mothblocks) [inline code/game/machinery/computer/communications.dm:772]: Why not `key_name_admin`?

Comment (ZephyrTFA) [reply code/game/machinery/computer/communications.dm:772]: key name admin shouldn't be used for logging purposes, given it turns it into an anchor to PM the user

Comment (Mothblocks) [reply code/game/machinery/computer/communications.dm:772]: Ahh I didn't see where this was okay

**Pull request merged**